include/aspect/particle/generator/probability_density_function.h

```diff
@@ -146,14 +146,14 @@ namespace aspect
            *
            * @param [in] particles_per_cell A vector with n_locally_owned_cells entries
            * that determines how many particles are generated in each cell.
-           * @param [in] local_start_id The starting ID to assign to generated particles of the local process.
+           * @param [in] first_particle_index The starting ID to assign to generated particles of the local process.
```

Comment:
I like that it recognized that the name is used here as well!

Reply:
Haha, you are giving it too much credit. This was edited by hand.